src/Microsoft.DotNet.ApiSymbolExtensions/Logging/ILog.cs

```diff
@@ -0,0 +1,14 @@
+﻿// Copyright (c) .NET Foundation and contributors. All rights reserved.
+// Licensed under the MIT license. See LICENSE file in the project root for full license information.
+
+using Microsoft.Build.Framework;
```

Comment:
I agree with Viktor, mixing frontends is not the best option. I believe, we must remove the usage of `Microsoft.Build.Framework` from here.